### PR TITLE
[T4A5][W11-B4]Tay Keming Justin

### PR DESCRIPTION
**Pull request by INCENDE**

### CODE REVIEW

**Comment by INCENDE** [comment]
Ready for review

File: src/seedu/addressbook/data/AddressBook.java [hunk 1]
```diff
@@ -143,4 +147,12 @@ public UniquePersonList getAllPersons() {
     public UniqueTagList getAllTags() {
         return new UniqueTagList(allTags);
     }
+    
+    /**
+     * Adds changes of tags that has been made to {@link #taggings}
+     */
+    public void addChangedTag(Person person, Tag tag, boolean isAdd)
+    {
```
Comment (okkhoy) [inline src/seedu/addressbook/data/AddressBook.java:155]: the brace should come immediately after the method name, not next line.
recollect the coding standards..

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
```
Comment (okkhoy) [inline src/seedu/addressbook/data/tag/Tagging.java:4]: Please include a header comment to indicate what this interface/class/method does?

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+	if (isAdd) {
+	    builder.append(TAG_ADD);
+	} else {
+	    builder.append(TAG_DELETE);
+	}
+	builder.append(" ");
+	builder.append(person.getName());
+	builder.append(" ");
+	builder.append("[");
+	builder.append(tag.tagName);
```
Comment (okkhoy) [inline src/seedu/addressbook/data/tag/Tagging.java:31]: you can use the getTag method here, which returns a formatted tag directly

**Comment by okkhoy** [comment]
@INCENDE 
Some comments added. Please ack & close the PR after reading comments.

**Pull request closed without merging**